docs/tutorials/03_sensor_calibration.md

```diff
@@ -0,0 +1,76 @@
+# 3. Sensor Calibration
+
+Estimate your intrinsic/extrinsic parameters on your sensor system using [tier4/calibration_tools](https://github.com/tier4/CalibrationTools).
+
+**NOTE: Perform the following tasks on an x86-based ECU**.
```

Comment:
```suggestion
**NOTE: Perform the following tasks on the x86-based ECU**.
```
the might be better since the computer was mentioned in the previous steps.